src/styles/product-cart.scss

```diff
@@ -0,0 +1,140 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
```

Comment:
1. You are setting styles for `body` in several places
2. Better idea is to have a dedicated file for that